src/Modules/Meetings/Tests/UnitTests/MeetingComments/MeetingCommentTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.Comments.Events;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingComments;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingComments.Rules;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.Meetings;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.Members;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.UnitTests.SeedWork;
+using NUnit.Framework;
+
+namespace CompanyName.MyMeetings.Modules.Meetings.Domain.UnitTests.MeetingComments
+{
+    [TestFixture]
+    public class MeetingCommentTests : TestBase
+    {
+        [Test]
+        public void CreateMeetingComment_IsSuccessful()
+        {
+            var meetingComment = MeetingComment.Create(
```

Comment:
To change after marking Create as internal.

Reply:
move tests for add comment logic to meeting tests